src/main/kotlin/org/jitsi/rtp/rtcp/rtcpfb/transport_layer_fb/tcc/RtcpFbTccPacket.kt

```diff
@@ -37,13 +39,17 @@ import org.jitsi.rtp.util.get3BytesAsInt
 import org.jitsi.rtp.util.getByteAsInt
 import org.jitsi.rtp.util.getShortAsInt
 
+open class PacketReport(val seqNum: Int)
+
+class UnreceivedPacketReport(seqNum: Int) : PacketReport(seqNum)
+
 // Size in bytes of a delta time in rtcp packet.
 // Valid values are 0 (packet wasn't received), 1 or 2.
 typealias DeltaSize = Int
 
-class ReceivedPacket(val seqNum: Int, val deltaTicks: Short) {
-    operator fun component1(): Int = seqNum
-    operator fun component2(): Short = deltaTicks
+class ReceivedPacketReport(seqNum: Int, val deltaTicks: Short) : PacketReport(seqNum) {
+    val deltaTicksDuration: Duration
```

Comment:
i wonder if this should just be named `deltaDuration`,  then it complements `deltaTicks` ?